x-pack/solutions/security/plugins/security_solution/server/lib/telemetry/async_sender.test.ts

```diff
@@ -1006,7 +1007,10 @@ describe('AsyncTelemetryEventsSender', () => {
 
   describe('ITelemetryEventsSender integration', () => {
     it('should send events using the async service', async () => {
-      const serviceV1 = new TelemetryEventsSender(loggingSystemMock.createLogger());
+      const serviceV1 = new TelemetryEventsSender(
+        loggingSystemMock.createLogger(),
+        {} as ExperimentalFeatures
```

Comment:
I don't see this argument defined in the `TelemetryEventsSender` constructor (the sender wasn't even modified by this PR), it shouldn't work 🤔 
Am I missing something?

Reply:
oh looks like I forgot to push that change upstream.

Reply:
5bb1e8409d6c1134a2cf2cbdf0c850d8ff65b7ca